README.md

```diff
@@ -16,7 +16,7 @@ they can play nice.
 
 # Installation
 
-## Step 1: Install Cellpose and Omnipose
+## Step 1: Install Cellpose and/or Omnipose
 
 ## Installation with Conda/Mamba
 We provide the following YAML file that installs Cellpose and omnipose in the same environment.
```

Comment:
Perhaps we should remove this whole part, as itàs bad for mac and the yaml is out of date.

Reply:
Makes sense, I think that with CUDA etc. installation is tricky enough that pointing people at the upstream docs might be easiest. Can just stress that an env with working cellpose is needed.